pkg/cmd/grafana/apiserver/cmd.go

```diff
@@ -30,6 +29,14 @@ func newCommandStartExampleAPIServer(o *APIServerOptions, stopCh <-chan struct{}
 			devAcknowledgementNotice,
 		Example: "grafana apiserver --runtime-config=example.grafana.app/v0alpha1=true",
 		RunE: func(c *cobra.Command, args []string) error {
+			if err := log.SetupConsoleLogger("debug"); err != nil {
```

Comment:
Debug hardcoded for now. Question if we should map verbosity level to grafana log level, i.e. if verbosity
- 0 = info
- 1 = warning
- 2 = error
- 3 = error
- >= 4 = debug

Based on https://github.com/kubernetes/klog/blob/007e661a007dacdf017ad9c1b53e9f96d32d4b7d/internal/severity/severity.go#L30-L36

Reply:
> We can probably figure this out in a separate PR

yeah, i think this is going to take some additional research. my understanding of the levels is limited, but it seems like they are closer to increasing the debug output